python/GafferUITest/BackgroundMethodTest.py

```diff
@@ -107,15 +107,15 @@ def __setTextPlug( self ) :
 
 			return self.__script["n"]["sum"]
 
-	class WaitingSlot( GafferTest.CapturingSlot ) :
+	class AwaitSetValueSlot( GafferTest.CapturingSlot ) :
```

Comment:
"Await" seems a bit misleading given that 3 has a specific `async/await` concept.

Reply:
As per our chat, have pushed an updated with new names/private namespace.

Reply:
Thanks Tom - LGTM.